apps/ui/src/common/pages/birth.tsx

```diff
@@ -190,10 +190,11 @@ const Page: FC<PageProps> = ({ location, signInHRef }) => {
                 name="dob"
                 prettyName="date of birth"
                 label={<h4>Date of birth</h4>}
+                hint="For example, 15/07/2018"
```

Comment:
as above - Part of another (unapproved) PR. I don't think this should be included.